### PR TITLE
Setup GCP commands

### PR DESCRIPTION
**Pull request by sergeymirzoyan1991**
Capture

### CODE REVIEW

File: default-filters/gcp/cloud-sql.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+and:
+  - resource: "launch-time"
+    op: "LessThanEqualTo"
```
Comment (marashlyansuren) [inline default-filters/gcp/cloud-sql.yaml:3]: GreaterThanEqualTo ???

File: src/index.ts [hunk 2]
```diff
@@ -2,14 +2,20 @@
...
+  CloudProvider,
+  Output,
+  OutputDirectory,
+  OutputFormats
+} from './constants'
 import CloudChiprCliProvider from './cli/cloud-chipr-cli-provider'
 import { LoggerHelper, OutputHelper } from './helpers'
 import moment from 'moment'
-require('dotenv').config({path: `${__dirname}/../.env`})
+require('dotenv').config({ path: `${__dirname}/../.env` })
```
Comment (marashlyansuren) [inline src/index.ts:14]: why do we need this?

File: src/responses/response-print.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import ResponseDecorator from './response-decorator'
+import { ProviderResource, Response } from '../../../engine'
+import { COLORS, Output } from '../constants'
+import { OutputHelper } from '../helpers'
+import chalk from 'chalk'
+
+export default class ResponsePrint {
```
Comment (marashlyansuren) [inline src/responses/response-print.ts:7]: So this is general for both AWS and GCP ?

File: default-filters/gcp/cloud-sql.yaml (hunk 1 above)
Comment (sergeymirzoyan1991) [reply default-filters/gcp/cloud-sql.yaml:3]: this will come on the follow-up PR

File: src/index.ts (hunk 2 above)
Comment (sergeymirzoyan1991) [reply src/index.ts:14]: this we had before, here I have just added spaces, but the idea behind this line is to enable c8r command to run from any directory of the system, and not only from its root dir.
not sure if I explained it clearly, let me know

File: src/responses/response-print.ts (hunk 3 above)
Comment (sergeymirzoyan1991) [reply src/responses/response-print.ts:7]: yes

**Review by marashlyansuren — APPROVED** [review]

**Pull request merged**